.github/workflows/tests-docker.yml

```diff
@@ -26,35 +26,29 @@ jobs:
         cp tests/test.env dev.env
         cat dev.env
         python3 wis2box-ctl.py config
-    - name: install loki plugin for docker
-      run: | 
-        docker plugin install grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
-        docker plugin ls
-    - name: build wis2box images and start containers ⚙️
+    - name: build wis2box
       run: |
         python3 wis2box-ctl.py build
+    - name: start containers ⚙️
+      run: |
         python3 wis2box-ctl.py start
-        sleep 5
-        python3 wis2box-ctl.py status
+        sleep 15
```

Comment:
To clarify, does this help the API / sequencing when waiting for ES?

Reply:
Yes